Modules/IO/TIFF/src/itkTIFFImageIO.cxx

```diff
@@ -559,9 +559,9 @@ TIFFImageIO::Write(const void * buffer)
 void
 TIFFImageIO::InternalWrite(const void * buffer)
 {
-  const auto * outPtr = (const char *)buffer;
+  const auto * outPtr = static_cast<const char *>(buffer);
 
-  unsigned int page, pages = 1;
+  size_t page, pages = 1;
```

Comment:
Why do you prefer `size_t` to `unsigned int`, for page and pages? I'm not sure but it looks like they should actually be 16-bit unsigned, for `TIFFTAG_PAGENUMBER`! http://www.libtiff.org/man/TIFFSetField.3t.html

Reply:
Did that because of the type of `m_Dimensions`, but I'll do the other way around.